### PR TITLE
Add an initial Hacktoberfest page

### PR DESCRIPTION
**Pull request by oleg-nenashev**

### CODE REVIEW

File: docs/events/README.md [hunk 1]
```diff
@@ -51,13 +51,15 @@ reach out to us on the `#contributing` channel.
 ### Hacktoberfest
 
 [Hacktoberfest](https://hacktoberfest.com/) is a month-long celebration of open-source projects, their maintainers, and the entire community of contributors.
-We plan to participate as an organization in October 2024.
 Many component maintainers from the Gradle ecosystem, e.g. the
 [Gradle plugin for Jenkins](https://plugins.jenkins.io/gradle),
 already include their projects in Hacktoberfest.
 
+In 2024, we plan to participate as an organization,
+and to offer nice special-edition swag to the key contributors.
```
Comment (lkasso) [inline docs/events/README.md:59]:
```suggestion
and to offer special-edition swag to lucky participants.
```

File: docs/events/README.md [hunk 2]
```diff
@@ -51,13 +51,15 @@ reach out to us on the `#contributing` channel.
...
+
+
 ## Be Informed
```
Comment (lkasso) [inline docs/events/README.md:62]:
```suggestion
## Stay Informed
```

File: docs/events/hacktoberfest/2024/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+title: "Gradle in Hacktoberfest 2024"
+description: >
+  Gradle participates in Hacktoberfest 2024!
+  Join us and contribute to the select projects,
+  and get custom swag!
+---
+
+[![Slack: hacktobest on the Gradle Community Slack](https://img.shields.io/badge/Slack-%23hacktoberfest-brightgreen?style=flat&logo=slack)](../../../contributing/community-slack.md)
+
+[Hacktoberfest](https://hacktoberfest.com/) is a month-long celebration of open-source projects, their maintainers, and the entire community of contributors.
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:11]:
```suggestion
[Hacktoberfest](https://hacktoberfest.com/) is a month-long celebration of open-source projects, their maintainers and contributors.
```

File: docs/events/hacktoberfest/2024/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+Many component maintainers from the Gradle ecosystem, e.g. the
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:12]:
```suggestion
Many component maintainers from the Gradle ecosystem, such as the
```

File: docs/events/hacktoberfest/2024/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+[Gradle plugin for Jenkins](https://plugins.jenkins.io/gradle),
+already include their projects in Hacktoberfest.
+
+In October, Gradle will participate in Hacktoberfest 2024!
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:16]:
```suggestion
This year, Gradle is participating in Hacktoberfest!
```

File: docs/events/hacktoberfest/2024/README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+This year, we are looking forward to offering a few featured projects, hosting online events for newcomer open-source contributors,
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:17]:
```suggestion
We are looking forward to featuring projects, hosting online events for new open-source contributors,
```

File: docs/events/hacktoberfest/2024/README.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+and sending custom Gradle goodies for those who contribute to the projects.
+
+## Featured Projects
+
+Here are a few projects we have in mind  for the event:
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:22]:
```suggestion
Here are a few projects we have in mind for the event:
```

File: docs/events/hacktoberfest/2024/README.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+* Making the Gradle plugins compatible with Gradle Configuration Cache ([list of plugin statuses on GitHub](https://github.com/gradle/gradle/issues/13490))
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:24]:
```suggestion
* Making Gradle plugins compatible with Gradle Configuration Cache ([list of plugin statuses on GitHub](https://github.com/gradle/gradle/issues/13490))
```

File: docs/events/hacktoberfest/2024/README.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+* Experimenting with Declarative Gradle and sharing feedback
+* Contributing new Gradle solutions to [Gradle Cookbook](https://community.gradle.org/cookbook/), and plugin documentation improvements
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:26]:
```suggestion
* Contributing new Gradle solutions to the [Gradle Cookbook](https://community.gradle.org/cookbook/)
```

Comment (lkasso) [reply docs/events/hacktoberfest/2024/README.md:26]: "plugin documentation improvements" isn't very specific.

File: docs/events/hacktoberfest/2024/README.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+* Learning Gradle Build Tool with free courses on [DPE University](https://dpeuniversity.gradle.com/app/catalog)
+
+More featured projects will be added soon,
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:29]:
```suggestion
More featured projects will be added soon.
```

File: docs/events/hacktoberfest/2024/README.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+maintainers are welcome to add their components if there are contributor guidelines
+and some issues that can be taken by contributors.
+
+## Swag
+
+In addition to common Hacktoberfest swag,
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:35]:
```suggestion
In addition to our Hacktoberfest swag,
```

File: docs/events/hacktoberfest/2024/README.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+
+    The details will be announced soon.
+
+## Events
+
+!!! info
+
+    Coming soon.
+    We will at least have an opening event,
+    and also a few presentations and office hours for newcomer contributors to open source.
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:49]:
```suggestion
    and also a few presentations and office hours for new contributors to open source.
```

File: docs/events/hacktoberfest/2024/README.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+
+## Be Informed
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:51]:
``` suggestion
## Stay Informed
```

File: docs/events/hacktoberfest/2024/README.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+Join the `#hacktoberfest` channel on the [Gradle Communbity Slack](../../../contributing/community-slack.md)
+where we will be posting the key updates and answering any questions.
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:54]:
```suggestion
where we will be posting key updates and answering questions.
```

File: docs/events/hacktoberfest/2024/README.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+Also, subscribe to the [newsletter](https://newsletter.gradle.org/) for updates
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:55]:
```suggestion
Subscribe to the [newsletter](https://newsletter.gradle.org/) for updates
```

File: docs/events/hacktoberfest/2024/README.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+about the upcoming community events and mentorship programs.
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:56]:
```suggestion
about upcoming community events and mentorship programs.
```

File: docs/events/hacktoberfest/2024/README.md [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+
+Here are a few projects we have in mind  for the event:
+
+* Making the Gradle plugins compatible with Gradle Configuration Cache ([list of plugin statuses on GitHub](https://github.com/gradle/gradle/issues/13490))
+* Experimenting with Declarative Gradle and sharing feedback
+* Contributing new Gradle solutions to [Gradle Cookbook](https://community.gradle.org/cookbook/), and plugin documentation improvements
+* Learning Gradle Build Tool with free courses on [DPE University](https://dpeuniversity.gradle.com/app/catalog)
+
+More featured projects will be added soon,
+maintainers are welcome to add their components if there are contributor guidelines
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:30]:
```suggestion
Maintainers are welcome to add their project if they provide contributor guidelines
```

File: docs/events/hacktoberfest/2024/README.md [hunk 18]
```diff
@@ -0,0 +1,57 @@
...
+and some issues that can be taken by contributors.
```
Comment (lkasso) [inline docs/events/hacktoberfest/2024/README.md:31]:
```suggestion
and issues that can be hacked on by contributors.
```

File: docs/events/hacktoberfest/README.md [hunk 19]
```diff
@@ -0,0 +1,13 @@
+# Gradle in Hacktoberfest
+
+[Hacktoberfest](https://hacktoberfest.com/) is a month-long celebration of open-source projects, their maintainers, and the entire community of contributors.
```
Comment (lkasso) [inline docs/events/hacktoberfest/README.md:3]:
```suggestion
[Hacktoberfest](https://hacktoberfest.com/) is a month-long celebration of open-source projects, their maintainers, and the global community of contributors.
```

File: docs/events/hacktoberfest/README.md [hunk 20]
```diff
@@ -0,0 +1,13 @@
...
+Many component maintainers from the Gradle ecosystem, e.g. the
```
Comment (lkasso) [inline docs/events/hacktoberfest/README.md:4]:
```suggestion
Many maintainers within the Gradle ecosystem, such as those behind the
```

File: docs/events/hacktoberfest/README.md [hunk 21]
```diff
@@ -0,0 +1,13 @@
...
+[Gradle plugin for Jenkins](https://plugins.jenkins.io/gradle),
+already include their projects in Hacktoberfest.
+
+In 2024, we plan to participate as an organization,
```
Comment (lkasso) [inline docs/events/hacktoberfest/README.md:8]:
```suggestion
In 2024, we’re excited to join as an organization```

File: docs/events/hacktoberfest/README.md [hunk 22]
```diff
@@ -0,0 +1,13 @@
...
+and to offer nice special-edition swag to the key contributors.
```
Comment (lkasso) [inline docs/events/hacktoberfest/README.md:9]:
```suggestion
and will be offering special-edition swag to key contributors.
```

**Review by lkasso — CHANGES_REQUESTED** [review]

File: docs/events/README.md (hunk 1 above)
Comment (oleg-nenashev) [reply docs/events/README.md:59]: No luck involved there

**Comment by oleg-nenashev** [comment]
@lkasso merged it to have content in place. Happy to address comments incrementally

**Comment by lkasso** [comment]
All good!

**Pull request merged**